packages/ui-library/src/components/transaction-receipt/index.tsx

```diff
@@ -307,7 +307,7 @@ const ERC20ApprovalTransactionReceipt = ({ transaction }: { transaction: ERC20Ap
   return (
     <>
       <StyledSectionContent>
-        <Typography variant="bodySmallLabel">
+        <Typography variant="labelRegular">
           <FormattedMessage description="TransactionReceipt-transactionAmountSent" defaultMessage="Amount approved" />
         </Typography>
         <Typography variant="bodySmallBold" sx={{ display: 'flex', alignItems: 'center', gap: spacing(2) }}>
```

Comment:
This all should be bodyBold right?

Reply:
I'm seeing 14px in figma, so bodySmallBold seems fine